frontend/src/app/components/search-form/search-form.component.ts

```diff
@@ -56,7 +57,9 @@ export class SearchFormComponent implements OnInit {
     private apiService: ApiService,
     private relativeUrlPipe: RelativeUrlPipe,
     private elementRef: ElementRef,
-  ) { }
+  ) {
+    this.searchAutofocus = ['/', '/mining', '/lightning'].includes(this.router.url);
```

Comment:
This probably doesn't work with languages added `/fr/`

Reply:
Replaced this approach by leveraging the state service instead